docs/esa/jwst/jwst.rst

```diff
@@ -104,23 +104,22 @@ service degradation.
   >>> width=u.Quantity(5, u.deg)
   >>> height=u.Quantity(5, u.deg)
   >>> r=Jwst.query_region(coordinate=coord, width=width, height=height)
-  >>> r
```

Comment:
was this deletion intentional (and many similar ones later)?  I believe you need this to get the printout below

Reply:
Yes, I did not see it in other modules, that is why I removed them. But I can include them again.

Reply:
Ok, done!

Reply:
I think it's a bit of mixed bag for the other modules. In general we tend to show the result of a query, but in some cases it's not very deterministic, so then we don't, or skip their doctest. 

Either way, I think it's better to avoid single letter variables everywhere, especially in the documentation and call everything something meaningful (just as you did with `width` and `height`, so what about using `result` or something similar?)